tools/importer/rules/marquee.js

```diff
@@ -0,0 +1,104 @@
+/* global WebImporter */
+export default function createMarqueeBlocks(main, document) {
+  const marquees = document.querySelectorAll('#marquee');
+
+  // fast return
+  if (!marquees) {
+    return;
+  }
+
+  marquees.forEach((marquee) => {
+    // Create table for marquee
+    const cells = [['Marquee']];
+
+    // Check if marquee has background video
+    if (marquee.querySelector('.has-video.dexter-Background')) {
+      const backgroundVideoURL = marquee.querySelector('.has-video.dexter-Background').querySelector('video').querySelector('source').getAttribute('src');
```

Comment:
Really ?

Reply:
Done, reduced the chaining and selection of element.